Lib/pathlib/_local.py

```diff
@@ -824,7 +823,9 @@ def rmdir(self):
         """
         os.rmdir(self)
 
-    _rmtree = shutil.rmtree
+    def _rmtree(self):
+        import shutil
+        shutil.rmtree(path=self)
```

Comment:
Can we remove this entirely?

```suggestion
```

I don't see it used anywhere in this file, or elsewhere for that matter.

Reply:
I believe it is overwriting a method on the base class.

Reply:
Yeah, it's overwriting `PathBase._rmtree()`. Using `shutil.rmtree()` is faster because it doesn't need to instantiate `Path` objects for the entire tree.